src/main/java/org/jabref/logic/importer/fetcher/GrobidCitationFetcher.java

```diff
@@ -72,4 +74,8 @@ private static String parseUsingGrobid(String plainText) throws FetcherException
     public String getName() {
         return "GROBID";
     }
+
+    public ArrayList<String> getFailedEntries() {
```

Comment:
Also return types should be interfaces as well:
https://www.quora.com/What-is-the-use-of-having-an-interface-as-a-return-type-in-Java